README.md

```diff
@@ -19,6 +19,11 @@ SMTP_SECURE=TRUE
 SMTP_FROM_MAIL=
 SMTP_USER=
 SMTP_PASS=
+# the URL where the dashboard is deployed (e.g. https://dashboard.kimai.owncloud.com)
```

Comment:
Be more generic and don't use "ownCloud" in the code and documentation e.g. `dashboard.kimai.example.com`